### DISCUSSION TITLE
Are there any good examples or tutorials out there for making Lightning Apps?

### DISCUSSION THREAD

**Discussion by Keiku** [Lightning-AI/pytorch-lightning#15255] [Lightning App API: LightningApp, LightningFlow, LightningWork] [answered]
I didn't really understand how the following App was made. Is there a good example or tutorial somewhere?

Lightning App Gallery https://lightning.ai/apps

**Comment by rohitgr7** [comment, marked as answer]
to get started you can look at: https://lightning.ai/lightning-docs/#get-started
and once you are familiar with the core structure, I'd suggest going over this tutorial: https://lightning.ai/lightning-docs/get_started/lightning_apps_intro.html

later on once you have good understanding of the components, you can pick any project you like from the gallary and explore the code-base and try to re-build it.

**Reply by Keiku** [reply]
Thank you for your answer. I will check the link.
